packages/e2e-playwright/src/templates/test-watcher.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * Playwright does not yet offer a means to run the tests in a watch mode.
```

Comment:
It is possible to pass this env variable to achieve it: `PWTEST_WATCH=1`
https://github.com/microsoft/playwright/issues/21960

Reply:
Alright, thanks. I already got rid of that file. It was part of an early iteration. There's also a button when running in UI mode that can be used to activate/deactivate watch mode. I'll push the changes soon.